assets/js/components/community/overrides/FileModificationModalMessage.js

```diff
@@ -0,0 +1,37 @@
+// This file is part of Zenodo.
+// Copyright (C) 2025 CERN.
+//
+// Zenodo is free software; you can redistribute it
+// and/or modify it under the terms of the GNU General Public License as
+// published by the Free Software Foundation; either version 2 of the
+// License, or (at your option) any later version.
+//
+// Zenodo is distributed in the hope that it will be
+// useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Zenodo; if not, write to the
+// Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+// MA 02111-1307, USA.
+//
+// In applying this license, CERN does not
+// waive the privileges and immunities granted to it by virtue of its status
+// as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+import React from "react";
+import { ModalContent } from "semantic-ui-react";
+
+const FileModificationModalMessage = () => {
+  return (
+    <ModalContent>
+      <p>
+        If you need to modify files in a deposit that was published more than 30 days ago, please contact our <a target="_blank" href="/support">support line</a>.
```

Comment:
```suggestion
        If you need to modify files in a record that was published more than 30 days ago, please contact our <a target="_blank" href="/support">support line</a>.
```
I think deposit is a bit dated